InformationScripting/src/parsing/SimpleQueryParser.cpp

```diff
@@ -71,14 +71,33 @@ QueryNode* SimpleQueryParser::parseAny(const QString& queryString, int& index)
 		if (index < queryString.length())
 			ch = queryString[index];
 	}
-	else query = new CommandNode(commandString); // Create the command. Note that it could be an empty string.
+	else
+	{
+		// Create the command. Note that it could be an empty string.
+		auto parts = commandString.split(" ");
```

Comment:
Does this work well, particularly when you type spaces and stuff? have you checked it with a few different cases?

Now that we have this feature it would be nice if we highlight commands and arguments in different ways. Commands should look more like keywords. Don't use the predefined styles however as those assume a light prompt. We'll need lighter font colors to better work with the new dark prompt.